wandb/apis/importers/wandb.py

```
@@ -0,0 +1,1133 @@
+            seen = seen | arts
+
+    def import_artifact_sequence(
+        self, sequence: ArtifactSequence, config: Optional[ImportConfig] = None
+    ) -> None:
+        """Import one artifact sequence.
+
+        Use `config` to specify alternate settings like where the artifact sequence should be uploaded
+        """
+        if config is None:
+            config = ImportConfig()
+
+        settings_override = {
+            "api_key": self.dst_api_key,
+            "base_url": self.dst_base_url,
+            "resumed": True,
+        }
+
+        send_manager_config = internal.SendManagerConfig(
+            log_artifacts=True,
+            debug_mode=config.debug_mode,
+        )
+
+        sequence = list(sequence)
+        placeholder_run = None
+        for s in sequence:
+            placeholder_run = s.logged_by()
+            if placeholder_run is not None:
+                break
+
+        if placeholder_run is None:
+            wandb_logger.error(
+                "Error finding placeholder run",
+                extra={
+                    "entity": "",
+                    "project": "",
+                    "run_id": "",
+                },
+            )
+            return
+
+        # instead of uploading placeholders 1 by 1,
+        # just upload the entire batch of placeholders in one run update
+        groups_of_artifacts = list(fill_with_dummy_arts(sequence))
+        art = groups_of_artifacts[0][0]
+        _type = art.type
+        name, *_ = art.name.split(":v")
+
+        task = progress.subtask_pbar.add_task(
+            f"Artifact Sequence ({_type}/{name})", total=len(groups_of_artifacts)
+        )
+        for group in groups_of_artifacts:
+            art = group[0]
+            if art.description == ART_SEQUENCE_DUMMY_DESCRIPTION:
+                run = WandbRun(placeholder_run)
+            else:
+                wandb_run = art.logged_by()
+                if wandb_run is None:
+                    continue
```

This is a legit case, we shouldn't silently ignore it.